### PR TITLE
discord pluign Makefile + Dockerfile

### PR DESCRIPTION
**Pull request by hugoShaka**
This PR adds the discord plugin main executable, Makefile and Dockerfile.

The main Makefile limitation is that you have to pass VERSION=XXX when invoking the make targets, I wanted to reuse the version from the root Makefile but it proved a bit difficult so I took this shortcut. When designing the pipeline you might want to fix this.

### CODE REVIEW

**Review by tigrato — APPROVED** [review]

File: integrations/kube-agent-updater/Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ ARG TARGETARCH
 # Build the program. We rely on golang's cross-compilation capabilities for multiarch building.
 RUN echo "Targeting $TARGETOS/$TARGETARCH" && \
     GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_ENABLED=0 \
-    go build -a -o /go/bin/teleport-kube-agent-updater github.com/gravitational/teleport/integrations/kube-agent-updater/cmd/teleport-kube-agent-updater
+    go build -a -ldflags "-w -s" -o /go/bin/teleport-kube-agent-updater github.com/gravitational/teleport/integrations/kube-agent-updater/cmd/teleport-kube-agent-updater
```
Comment (strideynet) [inline integrations/kube-agent-updater/Dockerfile:28]: Is this change intended to be in this PR ?

**Review by strideynet — APPROVED** [review]

File: integrations/kube-agent-updater/Dockerfile (hunk 1 above)
Comment (hugoShaka) [reply integrations/kube-agent-updater/Dockerfile]: yes but I reverted it, I will consolidate this later
